bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx88-en.html

```diff
@@ -0,0 +1,108 @@
+          <source type="video/webm" src="https://www-dev.allizom.org/media/img/firefox/whatsnew/whatsnew88/video-alt.d54d820f7ebda1b63be2c4a084b16782.webm" media="screen and (prefers-color-scheme: light)">
+        </video>
+      {% else %}
+        <img class="wnp-main-image" src="{{ static('img/firefox/whatsnew/whatsnew84/vpn-logo.svg') }}" width="120" alt="">
+      {% endif %}
+      <h1 class="wnp-main-title">Change your device’s location for <strong>next-level privacy</strong></h1>
+      <p class="wnp-main-tagline"><strong>Mozilla VPN</strong> (Virtual Private Network) protects your entire internet connection on your computer, your tablet, and even your phone. For even more security, it comes with a 30-day money back guarantee.</p>
+
+      <div class="wnp-main-cta">
+        <a href="{{ url('products.vpn.landing') }}{{ landing_source }}" class="mzp-c-button mzp-t-product mzp-t-lg" rel="external" data-cta-text="Get Mozilla VPN Now" data-cta-type="button">Get Mozilla VPN Now</a>
+      </div>
+    </div>
+  </section>
+
+  <section class="wnp-content-extra">
+    <div class="mzp-l-content mzp-t-content-xl">
+      <div class="wnp-platforms">
+        <h2>Available now for Windows, Mac, Android, iOS and Linux.</h2>
+        {% include 'firefox/whatsnew/includes/fx88/logo-windows.svg' %}
+        {% include 'firefox/whatsnew/includes/fx88/logo-linux.svg' %}
+        {% include 'firefox/whatsnew/includes/fx88/logo-apple.svg' %}
+        {% include 'firefox/whatsnew/includes/fx88/logo-android.svg' %}
+      </div>
+
+      <div class="l-columns-two">
+
+        <div class="c-picto-block">
+          <div class="c-picto-block-image">
+            {% include 'firefox/whatsnew/includes/fx88/illo-vpn-speed.svg' %}
+          </div>
+          <h3 class="c-picto-block-title">It’s fast, very fast</h3>
+          <div class="c-picto-block-body">
+            <p>Our WireGuard® powered servers are built for speed so you can stream, download, and game as usual.</p>
+          </div>
+        </div>
+
+        <div class="c-picto-block">
+          <div class="c-picto-block-image">
+            {% include 'firefox/whatsnew/includes/fx88/illo-vpn-lock.svg' %}
+          </div>
+          <h3 class="c-picto-block-title">Internet without a trace</h3>
+          <div class="c-picto-block-body">
+            <p>We never log, track, or share your network data — your online activity won’t leave a footprint on our VPN servers.</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <aside class="mzp-l-content c-utilities">
+    {% set releasenotes_url = url('firefox.releases.index') if not version else '/firefox/%s/releasenotes/'|format(version) %}
+    <p>{{ ftl('whatsnew-release-notes', url=releasenotes_url) }}</p>
+  </aside>
+
+</main>
+{% endblock %}
+
+{% block js %}
+  {{ js_bundle('firefox_whatsnew_88_en') }}
```

Comment:
If we put the traffic cop stuff into a separate bundle, we can add another call here inside a switch. Something like:
```
  {% if switch('firefox-whatsnew-88-en-experiment') %}
    {{ js_bundle('firefox_whatsnew_88_en_experiment') }}
  {% endif %}
```

Reply:
Should both bundles include the `up-to-date.js` so that I only load one JS bundle?

Reply:
No, only the main page bundle includes up-to-date and the bit for the easter egg. The experiment bundle is only the traffic cop code to do the redirects.